Sources/Plasma/PubUtilLib/plAudio/plAudioSystem.cpp

```diff
@@ -41,12 +41,7 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 *==LICENSE==*/
 #include "HeadSpin.h"
 #include <al.h>
-#ifdef USE_EFX
-#   include <efx.h>
-#endif
```

Comment:
To slightly complicate things, there are some OpenAL implementations (Apple) that don't ship the EFX header, which is why this was conditionally detected.

All of our builds use vcpkg which uses openal-soft, which includes EFX, but it would be nice to still gracefully fall back to no effects if EFX can't be found

Reply:
Ah, that's unfortunate.  I did this assuming we'd be using the same library across the different platforms, where OpenAL-soft provides this for us without variability.  

It does complicate things, but I can see about re-introducing proper conditional support.

Reply:
If it turns out to be a mess, we're probably okay to mandate OpenAL-Soft.

It's just nice to fallback to system libraries on weird old systems where you don't feel like trying to compile extra dependencies 😛

Reply:
Yeah, it's a good idea.  I just assumed I could simplify that code a bit and took the opportunity to clean it out. ;)